clang/include/clang/Format/Format.h

```diff
@@ -4656,6 +4687,15 @@ struct FormatStyle {
   /// \version 8
   std::vector<std::string> StatementMacros;
 
+  /// Tablegen
+  bool TableGenAllowBreakBeforeInheritColon;
+  bool TableGenAllowBreakAfterInheritColon;
```

Comment:
Do you need both options to be enabled at the same time? Otherwise `BreakInheritanceList` should already be the option to check.

Reply:
Thank you for the information. I removed these options. Instead  BreakInheritanceList works well. There happens some changes, but seems acceptable.